modules/ROOT/pages/api-groups-creating-groups.adoc

```diff
@@ -0,0 +1,86 @@
+= Create API Groups and Add API Instances
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can bundle a set of APIs to enable ease-of-use and efficiency for your API consumers. To create a bundle, you first create an API Group and then add the relevant API instances to it. If you want your API Group to be discovered by consumers, you must publish it to Anypoint Exchange.
+
+When you complete the task of creating an API Group with the help of this topic, you would have created an API Group, the very first version of that group, and the very first group instance. Additionally, you would also have added API instances to the API Group instance.
+
+You can add API instances to the API Group instance based on the environment (Production, Sandbox, or others) in which you create the API Group instance. For example, if you created a API Group instance in Production environment, you can add API instances to the API Group instance only from Production.
+
+You create API Groups using the following steps:
+
+*Step 1:* <<create-api-groups,Create an API Group>>
+
+The first step to creating an API Group that can be used by consumers, is create an API Group. 
+
+*Step 2:* <<adding-api-instances-to-groups,Add API instances to the new API Group>>
+
+After you create the API Group, you then add API instances to this group for the first time.
+
+== Task Prerequisites
+
+Before you begin, you must have:
+
+* At least one API instance in your organization or business group.
+* Organization Administrators permissions to access all the business groups and environments in which your API instances are located. 
```

Comment:
We should specify somewhere about the group administrator permission

Reply:
I don't found a specific doc talking about the permission but I found this:
```
The “Group Administrator” permission in a certain business group gives view access 
to all of the APIs of that business group so he can create groups using APIs from 
that organization. 
```

Please remember that this is the only permission that allows the users to take actions over groups (creating them, manage tiers and contracts, etc). And the scope of the permission is at BG level (not environment).

Reply:
Done! Thanks!